react/constants/SearchHelpers.js

```diff
@@ -67,6 +83,11 @@ export function getPagesArgs(opt, queryArg, mapArg, orderBy, isUnselectLink, typ
   const page = 'store/search'
   const params = { term: pathValues[0] }
   const Q = pathValues.splice(1).join(',') || undefined
-  const queryString = QueryString.stringify({ map, page: 1, O: orderBy, Q })
+  const queryString = QueryString.stringify({
+    map,
+    page: pageNumber,
+    O: orderBy,
+    Q,
```

Comment:
rename to query

Reply:
This is something with the backend, I think... @ClaudivanFilho did this

Reply:
👍